ui/lib/apps/SlowQuery/components/List.tsx

```diff
@@ -64,6 +65,18 @@ function List() {
   useEffect(() => {
     async function getSlowQueryList() {
       setLoading(true)
+      const recentMins = searchOptions.timeRange.recent
+      if (recentMins > 0) {
+        // beginTime & endTime is fixed value,
+        // so update them to the time range from now
+        const now = dayjs().unix()
+        const beginTime = now - recentMins * 60
+        searchOptions.timeRange = {
+          recent: recentMins,
+          begin_time: beginTime,
+          end_time: now,
+        }
+      }
```

Comment:
I will move this part logic to TimeSelector and supply a util method likes `updateTimeRange()` later.